### DISCUSSION TITLE
Tools and CI Meeting - Date 6/3

### DISCUSSION THREAD

**Discussion by spbrogan** [tianocore/edk2#5723] [General]
# Status

Scheduled for 6/3 @ 4:30pm Pacific Time

# Agenda

Follow up with pain points with Pull Request and Edk2 dev process.
Prioritize discussion around useful automations:
  * Adding reviewers (code owners)
  * Labels (add bugs, infosec process support, security functional fix vs CVEs)
Github issue transition

# Groups.io meeting invite:

https://edk2.groups.io/g/devel/viewevent?repeatid=43269&eventid=2159616&calstart=2024-01-08

# Microsoft Teams meeting

Join on your computer or mobile app
[Click here to join the meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZDI2ZDg4NmMtMjI1My00MzI5LWFmYjAtMGQyNjUzNTBjZGYw%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%2223af6561-6e1c-450d-b917-d9d674eb3cb6%22%7d)
Or call in (audio only)
+1 425-616-0754,,330776518# United States, Seattle
Phone Conference ID: 330 776 518#
Find a local number | [Reset PIN](https://mysettings.lync.com/pstnconferencing)
[Learn More](https://aka.ms/JoinTeamsMeeting) | [Meeting options](https://teams.microsoft.com/meetingOptions/?organizerId=23af6561-6e1c-450d-b917-d9d674eb3cb6&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&threadId=19_meeting_ZDI2ZDg4NmMtMjI1My00MzI5LWFmYjAtMGQyNjUzNTBjZGYw@thread.v2&messageId=0&language=en-US)

**Comment by spbrogan** [comment]
quick meeting with no recording.  

Notes:
1. PR process going pretty well.  Some feedback on the list and some confusion around merge process (Note: it hasn't changed.  Maintainers should still use "push" label).  
2. Biggest issue is assignment of reviewers.  Plan is to evaluate generating codeowners from maintainers.txt file for now and then let github assign the maintainers as PR required reviewer.   Maintainer would then add reviewers if they wanted help.  
3. Labels for PRs.  Some discussion around the `impact: security` vs CVE.  Waiting on @vincent-j-zimmer and the infosec group for comment.  Expected Wednesday (after next infosec).  
4. Discussion about move to github issues for "new" issues.  Next step send out RFC with the idea and details about how issues would be processed and handled.  
5. Discussion about github merge queue to replace mergify and depend on fewer services.  Todo: test/eval capabilities

Next time i will record so AI can take better notes.  Sorry.

**Comment by Ashwin-Veeraiah** [comment]
Is there a meeting for 6/17. I would like to add a topic to it. 
I am not able to find the post the meeting to reply on it as per the notes on #2614.
